bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/PanelImpl.java

```diff
@@ -114,6 +114,13 @@ public Boolean isReadOnly() {
         return readOnly;
     }
 
+    @Override
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @Nullable
+    public boolean isWrapData() {
```

Comment:
Why do you want to expose this property  ? Please link the JIRA id in the comment

Reply:
This property is needed to check if panel is marked as wrapped. This is needed to create bindRefMap required in adobe sign submission.

Reply:
https://jira.corp.adobe.com/browse/FORMS-16446

Reply:
This is not part of crispr spec. Even if this property is required, it should be a custom property specific to AEM implementation

Reply:
I will add JsonIgnore on it